README.Rmd

```diff
@@ -37,12 +57,15 @@ These options generally reduce runtimes at the cost of the granularity of estima
 
 The documentation for [`estimate_infections`](https://epiforecasts.io/EpiNow2/reference/estimate_infections.html) provides examples of the implementation of the different options available. 
 
-Forecasting is also supported for the time-varying reproduction number, infections and reported cases using the same generative process approach as used for estimation.
+### Other functions
```

Comment:
```suggestion
### Other models
```

I don't know if this is clearer - another option would be to just have three top-level bullet points for `estimate_infections()`, `estimate_secondary()`, `estimate_truncation()`. The reason the `estimate_infections` model is presented as the "main" model throughout is mostly for historical reasons.

Reply:
I like the idea of having the models as top-level bullet points. @jamesmbaazam for info historical reasons are just that `estimate_infections` was first /only for a few months.

Reply:
> for info historical reasons are just that estimate_infections was first /only for a few months

I see now. Thanks for clarifying. I've restructured the section.